views/backend/dashboard.php

```diff
@@ -20,9 +19,9 @@
             ?>
                 <div class="col l4 m6 s12">
                     <div class="card">
-                        <div class="card-content <?= $model_dashboard->getColor($table,$colors) ?> ">
+                        <div class="card-content <?= getColor($table,$colors) ?> ">
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'getColor'.](https://app.codacy.com/manual/JeromeLacquemant/projet-5/pullRequest?prid=5379676)